docs/website/docs/general-usage/transformations/index.md

```diff
@@ -1,16 +1,27 @@
 ---
-title: Transforming loaded datasets
-description: Define SQL-based or mixed SQL + Python transformations on data that is **already** in your destination.
+title: Transformations 🧪
+description: Define Python-based or mixed SQL + Python transformations on data that is **already** in your destination.
 keywords: [transformation, dataset, sql, pipeline, ibis, arrow]
 ---
-
-# Transformations: reshape data you have already loaded
-
-:::danger
-The `dlt.transformation` decorator is currently in preview and while it may appear to work very well, it should not be considered stable. If you found this unlinked page and would like to give us feedback, please let us know in our Slack community.
+# Transformations 🧪: move data after loading
```

Comment:
I prefer the old title actually, moving is just a small part of wart transformations does imho.